### PR TITLE
Remove some duplicate code

### PR DESCRIPTION
**Pull request by psykzz**
Credits @DominikPanic

### CODE REVIEW

**Comment by DominikPanic** [comment]
Found out during the port of https://github.com/tgstation/TerraGov-Marine-Corps/pull/1495

File: code/modules/mob/living/silicon/robot/emote.dm [hunk 1]
```diff
@@ -2,10 +2,6 @@
 	mob_type_allowed_typecache = list(/mob/living/silicon)
```
Comment (DominikPanic) [inline code/modules/mob/living/silicon/robot/emote.dm]: Check the line above if you're reviewing.

**Comment by Jalleo** [comment]
Thats emotive.

File: code/modules/mob/living/silicon/robot/emote.dm (hunk 1 above)
Comment (psykzz) [reply code/modules/mob/living/silicon/robot/emote.dm]: I did actually remove that line, but git being git determined the diff was below.

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**